Flaky test_grid_grid test

This is on master

```
AssertionError: tensor(1.2561e-05, grad_fn=<NormBackward0>) not less than 1.2e-05
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "/mnt/btrfs/trunk-hg-fbcode-fbsource-47681637-1552662588/fbcode/buck-out/dev/gen/pytorch/gpytorch/test_gpytorch#binary,link-tree/test/kernels/test_grid_kernel.py", line 31, in test_grid_grid
    self.assertLess(torch.norm(grid_eval - actual_eval), 1.2e-5)
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 1208, in assertLess
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 670, in fail
    raise self.failureException(msg)

test_grid_grid (test.kernels.test_grid_kernel.TestGridKernel) ... FAIL

======================================================================
FAIL: test_grid_grid (test.kernels.test_grid_kernel.TestGridKernel)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/btrfs/trunk-hg-fbcode-fbsource-47681637-1552662588/fbcode/buck-out/dev/gen/pytorch/gpytorch/test_gpytorch#binary,link-tree/test/kernels/test_grid_kernel.py", line 31, in test_grid_grid
    self.assertLess(torch.norm(grid_eval - actual_eval), 1.2e-5)
AssertionError: tensor(1.2561e-05, grad_fn=<NormBackward0>) not less than 1.2e-05

----------------------------------------------------------------------
Ran 1 test in 0.078s

FAILED (failures=1)
```

--- a/test/kernels/test_grid_kernel.py
+++ b/test/kernels/test_grid_kernel.py
@@ -28,7 +28,7 @@ class TestGridKernel(unittest.TestCase):
         self.assertIsInstance(grid_covar, KroneckerProductLazyTensor)
         grid_eval = kernel(grid_data, grid_data).evaluate()
         actual_eval = base_kernel(grid_data, grid_data).evaluate()
-        self.assertLess(torch.norm(grid_eval - actual_eval), 1.2e-5)
+        self.assertLess(torch.norm(grid_eval - actual_eval), 2e-5)
 
     def test_nongrid_grid(self):
         base_kernel = RBFKernel()

